build.sh

```diff
@@ -40,12 +40,19 @@ build_one() {
                 */NotoSansDevanagariUI-MM.plist)
                     g="${g/UI/}"
                     f='Noto Sans Devanagari UI'
-                    fontmake -g "$g" --mti-source "$1" --family-name "$f"
                     ;;
                 *)
-                    fontmake -g "$g" --mti-source "$1"
+                    f=''
                     ;;
             esac
+            if [[ -n "$f" ]]; then
```

Comment:
So the sources for the Sans Deva and DevaUI are the same, it's just that the UI font's name is different?